client/my-sites/plans/jetpack-plans/faq/index.tsx

```diff
@@ -51,9 +67,10 @@ const JetpackFAQ: React.FC = () => {
 			<section className="jetpack-faq">
 				<h2 className="jetpack-faq__heading">{ translate( 'Frequently Asked Questions' ) }</h2>
 				<FoldableFAQ
-					id="priority-support"
+					id="getting-started"
```

Comment:
This had the same `id` as another FAQ, I assume it was mishap so I went ahead and changed this one to one that made sense